src/__tests__/homepage.test.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import {cleanup, render, screen} from '@testing-library/react';
+import HomePage from "../pages/Home/Home.page";
+import youtubeVideos from "../mocks/youtube-videos";
+
+describe('Home page Tests', () => {
+  afterEach(cleanup);
```

Comment:
This is [done automatically](https://testing-library.com/docs/react-testing-library/setup/#skipping-auto-cleanup) for you when using `Jest`.

Reply:
Hahaha I had read that but the anxiety prevented me from avoiding it. Thanks